src/helpers/files.ts

```diff
@@ -22,7 +22,7 @@ export async function getFileListing(
   projectRoot: string,
   args: UploaderArgs,
 ): Promise<string> {
-  return getAllFiles(projectRoot, projectRoot, args).join('')
+  return getAllFiles(projectRoot, projectRoot, args).join('\n')
```

Comment:
`getAllFiles()` adds a `\n` to all the files already in one of the branches but not the other.
Unless you drop `.map(file => `${file}\n`)` from that function, this will produce double LFs (`\n\n`).

This is why in my PR I've included a bit of refactoring: https://github.com/codecov/uploader/pull/413/files#diff-ffd5012178bb277cea8ca8ffeea7ea4ccfc993570bd350f4a8a38c045422b271L271.

Reply:
Good catch!